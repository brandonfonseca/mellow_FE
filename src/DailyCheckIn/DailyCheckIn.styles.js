import styled from 'styled-components'

export const DailyCheckInWrapper = styled.div`
    display: flex;
    flex-direction: column;
    align-content: center;
`
