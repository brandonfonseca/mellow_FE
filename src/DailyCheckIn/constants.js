import Immutable from 'immutable'
import {
  verySadLogo, sadLogo, mehLogo, happyLogo, veryHappyLogo,
  waterZero, waterOne, waterTwo, waterThree, waterFour, waterFive, waterSix, waterSeven, waterEight,
  appleOne, appleTwo, appleThree, appleFour, appleFive,
  activityLow, activityMedium, activityHigh,
  sleepZero, sleepOne, sleepTwo, sleepThree, sleepFour, sleepFive, sleepSix, sleepSeven, sleepEight,
  thumbDown, thumbUp
} from '../Logos'

export const MOOD = 'mood'
export const WATER = 'water'
export const NUTRITION = 'nutrition'
export const ACTIVITY = 'activity'
export const SLEEP = 'sleep'
export const MEDITATE = 'meditate'

export const moodOptions = Immutable.fromJS({
  'Very Sad': [verySadLogo, 'mood1'],
  Sad: [sadLogo, 'mood2'],
  Meh: [mehLogo, 'mood3'],
  Happy: [happyLogo, 'mood4'],
  'Very Happy': [veryHappyLogo, 'mood5']
})

export const waterOptions = Immutable.fromJS({
  0: [waterZero, 'water0'],
  1: [waterOne, 'water1'],
  2: [waterTwo, 'water2'],
  3: [waterThree, 'water3'],
  4: [waterFour, 'water4'],
  5: [waterFive, 'water5'],
  6: [waterSix, 'water6'],
  7: [waterSeven, 'water7'],
  8: [waterEight, 'water8']
})

export const nutritionOptions = Immutable.fromJS({
  1: [appleOne, 'nutrition1'],
  2: [appleTwo, 'nutrition2'],
  3: [appleThree, 'nutrition3'],
  4: [appleFour, 'nutrition4'],
  5: [appleFive, 'nutrition5']
})

export const activityOptions = Immutable.fromJS({
  'Low Activity': [activityLow, 'activity1'],
  'Medium Activity': [activityMedium, 'activity2'],
  'High Activity': [activityHigh, 'activity3']
})

export const sleepOptions = Immutable.fromJS({
  0: [sleepZero, 'sleep0'],
  1: [sleepOne, 'sleep1'],
  2: [sleepTwo, 'sleep2'],
  3: [sleepThree, 'sleep3'],
  4: [sleepFour, 'sleep4'],
  5: [sleepFive, 'sleep5'],
  6: [sleepSix, 'sleep6'],
  7: [sleepSeven, 'sleep7'],
  8: [sleepEight, 'sleep8']
})

export const meditateOptions = Immutable.fromJS({
  No: [thumbDown, 'meditate0'],
  Yes: [thumbUp, 'meditate1']
})
