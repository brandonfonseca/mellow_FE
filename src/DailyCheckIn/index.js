import React from 'react'
import OptionsCard from '../CommonComponents/OptionsCard'
import moodLogo from '../Logos/CardHeaderLogos/mood.svg'
import waterLogo from '../Logos/CardHeaderLogos/water.svg'
import appleLogo from '../Logos/CardHeaderLogos/apple.svg'
import weightLogo from '../Logos/CardHeaderLogos/weight.svg'
import moonLogo from '../Logos/CardHeaderLogos/moon.svg'
import flowerLogo from '../Logos/CardHeaderLogos/flower.svg'

import {
  MOOD, WATER, NUTRITION, ACTIVITY, SLEEP, MEDITATE,
  moodOptions, waterOptions, nutritionOptions, activityOptions,
  sleepOptions, meditateOptions
} from './constants'
import { DailyCheckInWrapper } from './DailyCheckIn.styles'

const DailyCheckIn = () => {
  return (
    <DailyCheckInWrapper>
      <OptionsCard title={MOOD} prompt={'How was your overall mood today?'}
        logo={moodLogo} options={moodOptions}/>
      <OptionsCard title={WATER} prompt={'How many cups of water did you drink today?'}
        logo={waterLogo} options={waterOptions}/>
      <OptionsCard title={NUTRITION} prompt={'How would you rate the nutritional value of your food today?'}
        logo={appleLogo} options={nutritionOptions}/>
      <OptionsCard title={ACTIVITY} prompt={'How active were you today?'}
        logo={weightLogo} options={activityOptions} extraSpacing={true}/>
      <OptionsCard title={SLEEP} prompt={'How many hours of sleep did you get last night?'}
        logo={moonLogo} options={sleepOptions}/>
      <OptionsCard title={MEDITATE} prompt={'Did you meditate today?'}
        logo={flowerLogo} options={meditateOptions}/>
    </DailyCheckInWrapper>
  )
}

export default DailyCheckIn
