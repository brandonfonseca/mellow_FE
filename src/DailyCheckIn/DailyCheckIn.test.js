import React from 'react'
import { render } from '@testing-library/react'
import DailyCheckIn from '.'

describe('Daily Check-in tab', () => {
  it('renders the daily check-in items', () => {
    const wrapper = render(<DailyCheckIn/>)
    expect(wrapper).toMatchSnapshot()
  })
})
