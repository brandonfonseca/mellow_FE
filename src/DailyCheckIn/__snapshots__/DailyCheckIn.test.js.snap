// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Daily Check-in tab renders the daily check-in items 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-Axmtr kExLnP"
      >
        <div
          class="sc-AxjAm jLVhHd"
        >
          <div
            class="sc-AxirZ kLCRzb"
          >
            MOOD
          </div>
          <div
            class="sc-AxiKw gfdoYB"
          >
            How was your overall mood today?
          </div>
          <img
            class="sc-AxhCb jOIycJ"
            src="mood.svg"
          />
          <div
            class="sc-AxhUy knSSze"
          >
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="very-sad.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Very Sad
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sad.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Sad
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="meh.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Meh
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="happy.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Happy
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="very-happy.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Very Happy
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-AxjAm jLVhHd"
        >
          <div
            class="sc-AxirZ kLCRzb"
          >
            WATER
          </div>
          <div
            class="sc-AxiKw gfdoYB"
          >
            How many cups of water did you drink today?
          </div>
          <img
            class="sc-AxhCb jOIycJ"
            src="water.svg"
          />
          <div
            class="sc-AxhUy knSSze"
          >
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water0.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                0
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water1.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                1
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water2.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                2
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water3.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                3
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water4.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                4
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water5.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                5
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water6.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                6
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water7.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                7
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="water8.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                8
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-AxjAm jLVhHd"
        >
          <div
            class="sc-AxirZ kLCRzb"
          >
            NUTRITION
          </div>
          <div
            class="sc-AxiKw gfdoYB"
          >
            How would you rate the nutritional value of your food today?
          </div>
          <img
            class="sc-AxhCb jOIycJ"
            src="apple.svg"
          />
          <div
            class="sc-AxhUy knSSze"
          >
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="apple1.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                1
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="apple2.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                2
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="apple3.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                3
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="apple4.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                4
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="apple5.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                5
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-AxjAm jLVhHd"
        >
          <div
            class="sc-AxirZ kLCRzb"
          >
            ACTIVITY
          </div>
          <div
            class="sc-AxiKw gfdoYB"
          >
            How active were you today?
          </div>
          <img
            class="sc-AxhCb jOIycJ"
            src="apple.svg"
          />
          <div
            class="sc-AxhUy knSSze"
          >
            <div
              class="sc-AxgMl fuoSmu"
            >
              <img
                alt=""
                src="activity-low.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Low Activity
              </div>
            </div>
            <div
              class="sc-AxgMl fuoSmu"
            >
              <img
                alt=""
                src="activity-medium.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Medium Activity
              </div>
            </div>
            <div
              class="sc-AxgMl fuoSmu"
            >
              <img
                alt=""
                src="activity-high.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                High Activity
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-AxjAm jLVhHd"
        >
          <div
            class="sc-AxirZ kLCRzb"
          >
            SLEEP
          </div>
          <div
            class="sc-AxiKw gfdoYB"
          >
            How many hours of sleep did you get last night?
          </div>
          <img
            class="sc-AxhCb jOIycJ"
            src="apple.svg"
          />
          <div
            class="sc-AxhUy knSSze"
          >
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep0.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                0
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep1.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                1
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep2.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                2
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep3.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                3
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep4.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                4
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep5.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                5
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep6.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                6
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep7.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                7
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="sleep8.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                8
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-AxjAm jLVhHd"
        >
          <div
            class="sc-AxirZ kLCRzb"
          >
            MEDITATE
          </div>
          <div
            class="sc-AxiKw gfdoYB"
          >
            Did you meditate today?
          </div>
          <img
            class="sc-AxhCb jOIycJ"
            src="apple.svg"
          />
          <div
            class="sc-AxhUy knSSze"
          >
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="thumbDown.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                No
              </div>
            </div>
            <div
              class="sc-AxgMl egSYJi"
            >
              <img
                alt=""
                src="thumbUp.svg"
              />
              <div
                class="sc-AxheI cZAbpI"
              >
                Yes
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-Axmtr kExLnP"
    >
      <div
        class="sc-AxjAm jLVhHd"
      >
        <div
          class="sc-AxirZ kLCRzb"
        >
          MOOD
        </div>
        <div
          class="sc-AxiKw gfdoYB"
        >
          How was your overall mood today?
        </div>
        <img
          class="sc-AxhCb jOIycJ"
          src="mood.svg"
        />
        <div
          class="sc-AxhUy knSSze"
        >
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="very-sad.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Very Sad
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sad.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Sad
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="meh.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Meh
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="happy.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Happy
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="very-happy.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Very Happy
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-AxjAm jLVhHd"
      >
        <div
          class="sc-AxirZ kLCRzb"
        >
          WATER
        </div>
        <div
          class="sc-AxiKw gfdoYB"
        >
          How many cups of water did you drink today?
        </div>
        <img
          class="sc-AxhCb jOIycJ"
          src="water.svg"
        />
        <div
          class="sc-AxhUy knSSze"
        >
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water0.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              0
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water1.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              1
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water2.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              2
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water3.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              3
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water4.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              4
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water5.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              5
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water6.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              6
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water7.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              7
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="water8.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              8
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-AxjAm jLVhHd"
      >
        <div
          class="sc-AxirZ kLCRzb"
        >
          NUTRITION
        </div>
        <div
          class="sc-AxiKw gfdoYB"
        >
          How would you rate the nutritional value of your food today?
        </div>
        <img
          class="sc-AxhCb jOIycJ"
          src="apple.svg"
        />
        <div
          class="sc-AxhUy knSSze"
        >
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="apple1.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              1
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="apple2.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              2
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="apple3.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              3
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="apple4.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              4
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="apple5.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              5
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-AxjAm jLVhHd"
      >
        <div
          class="sc-AxirZ kLCRzb"
        >
          ACTIVITY
        </div>
        <div
          class="sc-AxiKw gfdoYB"
        >
          How active were you today?
        </div>
        <img
          class="sc-AxhCb jOIycJ"
          src="apple.svg"
        />
        <div
          class="sc-AxhUy knSSze"
        >
          <div
            class="sc-AxgMl fuoSmu"
          >
            <img
              alt=""
              src="activity-low.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Low Activity
            </div>
          </div>
          <div
            class="sc-AxgMl fuoSmu"
          >
            <img
              alt=""
              src="activity-medium.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Medium Activity
            </div>
          </div>
          <div
            class="sc-AxgMl fuoSmu"
          >
            <img
              alt=""
              src="activity-high.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              High Activity
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-AxjAm jLVhHd"
      >
        <div
          class="sc-AxirZ kLCRzb"
        >
          SLEEP
        </div>
        <div
          class="sc-AxiKw gfdoYB"
        >
          How many hours of sleep did you get last night?
        </div>
        <img
          class="sc-AxhCb jOIycJ"
          src="apple.svg"
        />
        <div
          class="sc-AxhUy knSSze"
        >
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep0.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              0
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep1.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              1
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep2.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              2
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep3.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              3
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep4.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              4
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep5.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              5
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep6.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              6
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep7.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              7
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="sleep8.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              8
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-AxjAm jLVhHd"
      >
        <div
          class="sc-AxirZ kLCRzb"
        >
          MEDITATE
        </div>
        <div
          class="sc-AxiKw gfdoYB"
        >
          Did you meditate today?
        </div>
        <img
          class="sc-AxhCb jOIycJ"
          src="apple.svg"
        />
        <div
          class="sc-AxhUy knSSze"
        >
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="thumbDown.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              No
            </div>
          </div>
          <div
            class="sc-AxgMl egSYJi"
          >
            <img
              alt=""
              src="thumbUp.svg"
            />
            <div
              class="sc-AxheI cZAbpI"
            >
              Yes
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
